{"name": "Unsuccessful login", "status": "passed", "steps": [{"name": "Error message is visible", "status": "passed", "start": 1737700609221, "stop": 1737700609221}], "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "'secret_sauce'"}, {"name": "driver", "value": "'Edge'"}], "start": 1737700608703, "stop": 1737700609298, "uuid": "e24a622c-094b-4bb6-a60f-edd851d77b9a", "historyId": "e664392ae8e96150bc26ed53191a48bb", "testCaseId": "b876685be5aab7257716ff999cc789c0", "fullName": "Tests.test_Login.TestLogin#test_1", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1120-982"}, {"name": "thread", "value": "1996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}